server/routes/clientRouter.js

```diff
@@ -2,9 +2,10 @@ import express from "express"
 
 const router = new express.Router()
 
-const clientRoutes = []
+const clientRoutes = [] 
```

Comment:
Make sure to update this with only the path needed for the first feature.  It's important to work slowly and only add what we need when we need it